github/git_refs.go

```diff
@@ -58,13 +61,60 @@ func (s *GitService) GetRef(ctx context.Context, owner string, repo string, ref
 
 	r := new(Reference)
 	resp, err := s.client.Do(ctx, req, r)
-	if err != nil {
+
```

Comment:
No need for a blank line here. It makes it harder to see that the `err` value is being handled on next line.